SU2_CFD/src/solvers/CNEMOEulerSolver.cpp

```diff
@@ -262,6 +264,58 @@ CNEMOEulerSolver::CNEMOEulerSolver(CGeometry *geometry, CConfig *config,
     }
   }
 
+
+  /*--- Correct normal directions of edges ---*/
+  for (unsigned long iMarker = 0; iMarker < geometry->GetnMarker(); iMarker++) {
+    if (config->GetMarker_All_KindBC(iMarker) == SYMMETRY_PLANE){
```

Comment:
We should probably move this to draft PR while you guys prototype the new symmetry implementation.
Then we can discuss how and where to implement the different changes without copy-pasting code and making sure the fix is compatible with other features of the code.